src/outputs/output_utils.hpp

```diff
@@ -47,7 +47,7 @@ namespace OutputUtils {
 // Helper struct containing some information about a variable
 struct VarInfo {
  public:
-  static constexpr int VNDIM = MAX_VARIABLE_DIMENSION;
+  static constexpr const int VNDIM = MAX_VARIABLE_DIMENSION;
```

Comment:
Isn't the extra `const` here redundant given that it is already `constexpr`?

Reply:
I think so.